windows_containers/index.adoc

```diff
@@ -0,0 +1,27 @@
+[id="windows-container-overview"]
+= Windows Container Support for OpenShift overview
+include::modules/common-attributes.adoc[]
+:context: windows-container-overview
+
+toc::[]
+
+Windows Container Support for OpenShift is a feature providing the ability to run Windows compute nodes in an {product-title} cluster. This is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With a Red Hat subscription, you can get support for running Windows workloads in {product-title}. For more information, see the xref:../windows_containers/windows-containers-release-notes-4-x.adoc#windows-containers-release-notes-4-x[release notes].
+
+For workloads including both Linux and Windows, {product-title} allows you to deploy Windows workloads running on Windows Server containers while also providing traditional Linux workloads hosted on {op-system-first} or {op-system-base-full}. For more information, see xref:../windows_containers/understanding-windows-container-workloads.adoc#understanding-windows-container-workloads[getting started with Windows container workloads].
+
+You need a WMCO to run Windows workloads in your cluster. The WMCO orchestrates the process of deploying and managing Windows workloads on a cluster. For more information, see xref:../windows_containers/enabling-windows-container-workloads.adoc#enabling-windows-container-workloads[how to enable Windows container workloads].
```

Comment:
s/a WMCO/the WMCO (To me, _a_ suggests there can be more than one; like _a node_.)

Reply:
acked